File: Features/Reservations/Resources/PassengerWriteResource.cs
using System;

namespace ShipCruises {

    public class PassengerWriteResource {

        public int Id { get; set; }
        public Guid? ReservationId { get; set; }
        public int OccupantId { get; set; }
        public int NationalityId { get; set; }
        public int GenderId { get; set; }
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string BirthDate { get; set; }
        public string SpecialCare { get; set; }
        public string Remarks { get; set; }
        public bool IsCheckedIn { get; set; }

    }

}

File: FrontEnd/src/app/features/reservations/classes/resources/form/passenger/passenger-write-resource.ts
import { Guid } from 'guid-typescript'

export class PassengerWriteResource {

    id: number
    reservationId: Guid
    occupantId: number
    nationalityId: number
    genderId: number
    lastname: string
    firstname: string
    birthdate: string
    specialCare: string
    remarks: string
    isCheckedIn: boolean

}